examples/gno.land/p/demo/grc/grc20/account_banker.gno

```diff
@@ -0,0 +1,70 @@
+package grc20
+
+import (
+	"std"
+)
+
+func PrevRealmBanker(b *Bank) *fnBanker {
+	return &fnBanker{
+		accountFn: func() std.Address {
+			caller := std.PrevRealm().Addr()
+			return caller
+		},
+		bank: b,
+	}
+}
+
+func ReadonlyBanker(b *Bank) *fnBanker {
+	return &fnBanker{
+		accountFn: nil,
+		bank:      b,
+	}
+}
+
+func AccountBanker(b *Bank, slug string) *fnBanker {
```

Comment:
I also don't intuitively understand this one (before reading the implementation). Ideas:

- Simply, "Sub-account" to identify this is a child account scoped with the address of the parent.
- Revolut uses the word "Pocket" to talk about sub-accounts. Not particularly enthusiastic about tihs but worth considering.
- I've seen other places call them `Vault`. I like this one.
- But we can also call it `Piggy` with reference to a [piggy bank](https://en.wikipedia.org/wiki/Piggy_bank).